lang/cs/Org.Apache.REEF.Tests/Functional/Failure/EvaluatorWithRunningTaskDelayedPoisonTest.cs

```diff
@@ -127,7 +114,7 @@ public void OnNext(IFailedEvaluator value)
 
                 Logger.Log(Level.Info, "Received all expected failed Tasks.");
 
-                const string expectedStr = "expected a single Context with Context ID " + ContextId + ".";
+                var expectedStr = "expected a single Context with Context ID " + ContextId + ".";
 
                 if (value.FailedContexts == null)
```

Comment:
I would assume the test control the timing. In reality, failed context can be null if Evaluator fails before Context is created.